src/main/java/frc/robot/subsystems/drive/ModuleIOSparkMax.java

```diff
@@ -89,9 +92,27 @@ public ModuleIOSparkMax(ModuleConfig config) {
     turnSparkMax.setPeriodicFramePeriod(PeriodicFrame.kStatus2, (int) (1000.0 / odometryFrequency));
     timestampQueue = SparkMaxOdometryThread.getInstance().makeTimestampQueue();
     drivePositionQueue =
-        SparkMaxOdometryThread.getInstance().registerSignal(driveEncoder::getPosition);
+        SparkMaxOdometryThread.getInstance()
+            .registerSignal(
```

Comment:
lots of duplicate code

Reply:
I mean they arent really the same. One is checking drive the other is checking turn. Not sure how you could simplify more